test/e2e/rmi_test.go

```diff
@@ -137,12 +137,12 @@ var _ = Describe("Podman rmi", func() {
 		Expect(session).Should(Exit(0))
 		Expect(len(session.OutputToStringArray())).To(Equal(1))
 
-		session = podmanTest.PodmanNoCache([]string{"images", "-q", "-a"})
+		session = podmanTest.PodmanNoCache([]string{"images", "--sort", "created", "--format", "{{.Id}}", "--all"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 		Expect(len(session.OutputToStringArray())).To(Equal(2),
 			"Output from 'podman images -q -a':'%s'", session.Out.Contents())
-		untaggedImg := session.OutputToStringArray()[0]
```

Comment:
I like the new test, but I'd like to keep the old and comment it out or make it a new test and skip for now until #6670 is fixed.

Reply:
#6670 must add tests to make sure that `-q` sorts correctly